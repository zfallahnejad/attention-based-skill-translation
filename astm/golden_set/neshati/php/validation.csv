118068,validation
741747,validation
1415724,validation
476,validation
250259,validation
208809,validation
1392379,validation
1317935,validation
50079,validation
345031,validation
31671,validation
367456,validation
1338292,validation
187606,validation
42139,validation
