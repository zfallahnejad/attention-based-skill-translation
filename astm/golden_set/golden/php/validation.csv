118068,12,16,0.42857142857142855
741747,11,11,0.5
1415724,11,6,0.6470588235294118
476,11,10,0.5238095238095238
250259,10,11,0.47619047619047616
208809,9,7,0.5625
50079,8,11,0.42105263157894735
345031,8,9,0.47058823529411764
1392379,8,2,0.8
1317935,8,7,0.5333333333333333
42139,7,9,0.4375
367456,7,9,0.4375
31671,7,8,0.4666666666666667
187606,7,8,0.4666666666666667
1338292,7,9,0.4375
